src/main/resources/application.yaml

```diff
@@ -18,9 +18,16 @@ management:
         include: health, info
 
 url:
-  fellesdatakatalog.api: ${RESOURCE_SERVICE_URI}
-  fellesdatakatalog.frontend: ${FDK_URI}
+  fellesdatakatalog:
+    api: ${RESOURCE_SERVICE_URI}
+    frontend: ${FDK_URI}
+  kudaf:
+    soknadApi: https://frontend-soknad-api.paas2.uninett.no
 
+
+springdoc:
+  swagger-ui:
+    path: /
```

Comment:
Legger spring doc til root. Vet ikke om noen har noen formeninger om hvor det bør ligge

Reply:
Endrer det i [Use default endpoint for swagger](https://github.com/fellesdatakatalog/access-request-api/pull/6/commits/bcd93b451f42cfb9e6ee70f7019d0afe0e020ccf)